src/components/value-list/value-list.tsx

```diff
@@ -335,7 +382,17 @@ export class ValueList<
     return type;
   }
 
+  updateScreenReaderText(assertiveText: string): void {
+    //code to update text to our aria-live span element
+    const screenReaderElement = this.el.shadowRoot.querySelector(".assistive-text");
+    screenReaderElement.textContent = assertiveText;
+  }
+
+  updateHandleAriaLabel(handleElement: HTMLSpanElement, assertiveText: string): void {
+    handleElement.ariaLabel = assertiveText;
```

Comment:
nitpick (and if changing to `aria-live="polite"`): rename `assertiveText` to `accessibleText`, which could help if we make changes down the road as well.